neurom/analysis/tests/test_morphmath.py

```diff
@@ -74,21 +74,82 @@ def test_point_dist():
     nt.ok_(dist==sqrt(3))
 
 
-def test_angle_3points():
+def test_angle_3points_half_pi():
+    orig = (0.0, 0.0, 0.0)
     vec1 = (1.0, 0.0, 0.0)
-    vec2 = (0.0, 1.0, 0.0)
-    orig = (0.0,0.0,0.0)
-    angle=angle_3points(orig, vec1, vec2)
-    nt.ok_(angle==pi/2.0)
+    vec2 = (0.0, 2.0, 0.0)
+    nt.ok_(angle_3points(orig, vec1, vec2)==pi/2.0)
+
+    vec2 = (0.0, 0.0, 3.0)
+    nt.ok_(angle_3points(orig, vec1, vec2)==pi/2.0)
+
+    vec2 = (0.0, 0.0, -3.0)
+    nt.ok_(angle_3points(orig, vec1, vec2)==pi/2.0)
+
+    vec1 = (0.0, 4.0, 0.0)
+    nt.ok_(angle_3points(orig, vec1, vec2)==pi/2.0)
+
+
+def test_angle_3points_quarter_pi():
+    orig = (0.0, 0.0, 0.0)
+    vec1 = (1.0, 0.0, 0.0)
+    vec2 = (2.0, 2.0, 0.0)
+    nt.assert_equal(angle_3points(orig, vec1, vec2), pi / 4.0)
```

Comment:
I also prefer eq_ to assert_equal, as that actually comes from unittest via a weird set of imports, fyi